Gemfile

```diff
@@ -70,7 +70,6 @@ group :test, :development do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'bullet'
   gem 'capybara'
-  gem 'devise'
```

Comment:
- [ ] It would be better if we implement the devise functionality. The commit messages show that our project "add devise" is reverted, meanwhile, the PR says "add devise". Let us kindly resolve this and submit another code review request